xbmc/pvr/epg/Epg.cpp

```diff
@@ -339,10 +339,13 @@ bool CPVREpg::Delete(const std::shared_ptr<CPVREpgDatabase>& database)
   }
 
   // delete own epg db entry
-  database->Delete(*this);
+  if (bCommit)
+    database->Delete(*this);
+  else
+    database->QueueDeleteEpgQuery(*this);
 
   // delete all tags for this epg from db
-  m_tags.Delete();
+  m_tags.Delete(bCommit);
```

Comment:
Same here. Will never be called with commit = true. Thus, rename the method and simplify its implementation.